mlir/lib/Dialect/Complex/IR/ComplexOps.cpp

```diff
@@ -369,6 +369,29 @@ OpFoldResult MulOp::fold(FoldAdaptor adaptor) {
   return {};
 }
 
+//===----------------------------------------------------------------------===//
+// DivOp
+//===----------------------------------------------------------------------===//
+
+OpFoldResult DivOp::fold(FoldAdaptor adaptor) {
+  auto constant = getRhs().getDefiningOp<ConstantOp>();
```

Comment:
The folder is supposed to read the constant from the provided adaptor.

Reply:
I updated the patch to use the adaptor accordingly. Could you take a look at this again when you get a chance?